pages/ordeal-bubble-chart.vue

```diff
@@ -0,0 +1,60 @@
+            <line x1="12" y1="16" x2="12" y2="12"></line><line x1="12" y1="8" x2="12" y2="8"></line>
+          </svg>
+        </span>
+      </div>
+
+    </div>
+    <div>
+      <iframe style="width: 80vw; height: 70vh; border: none;" src="https://query.wikidata.org/embed.html#%23Ordeal%20of%20the%20accused%20witches%20in%20the%20Survey%20of%20Scottish%20witchcraft%0A%0A%23defaultView%3ABubbleChart%0ASELECT%20(IF(BOUND(%3Fordeal)%2C%20%3FordealLabel%2C%20%22unknown%22)%20as%20%3Flabel)%20%3Fcount%0A%3Fordeal%20%3FordealLabel%20%0A%0AWHERE%0A%7B%0A%20%20%7B%0A%20%20%20%20SELECT%20%3Fordeal%20(COUNT(%3Fperson)%20AS%20%3Fcount)%20WHERE%20%7B%0A%20%20%20%20%20%20%3Fperson%20wdt%3AP4532%20%3Faccused%20.%0A%20%20%20%20%20%20OPTIONAL%20%0A%20%20%20%20%20%20%7B%0A%20%20%20%20%20%20%3Fperson%20wdt%3AP7209%20%3Fordeal.%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%20%20GROUP%20BY%20%3Fordeal%0A%20%20%7D%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0AORDER%20BY%20DESC(%3Fcount)%0ALIMIT%204000" referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups"></iframe>
+    </div>
+    <h1 class="text-sm md:text-xl lg:text-2xl">Trial Ordeal without Unknown Values</h1>
+    <iframe style="width: 80vw; height: 70vh; border: none;" src="https://query.wikidata.org/embed.html#%23ORDEAL%0A%23defaultView%3ABubbleChart%0ASELECT%20%3FordealLabel%20(COUNT(%3Ftrial)%20as%20%3Fcount)%20%0AWHERE%20%0A%7B%0A%20%20%3Fitem%20wdt%3AP4532%20%3Ftrial.%0A%20%0A%20%20%3Fitem%20wdt%3AP7209%20%3Fordeal.%20%0A%20%20%0A%20%20%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0AGROUP%20BY%20%3FordealLabel%0A%0AORDER%20BY%20DESC%20(%3Fcount)" referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups"></iframe>
+  </div>
+
+</template>
+
+<script>
+import Swal from 'sweetalert2';
+
+definePageMeta({
+   layout: 'default'
+ })
+
+ export default {
+   data: () => ({
+
+   }),
+   methods: {
+     showPageInfo(){
+       Swal.fire({
+         title: 'Ordeal Bubble Charts',
+         html: '<div>These bubble charts show the different trial ordeals that took place during the witchcraft trials. Trial by ordeal was a judicial practice in which the accused witches innocence was tested through an unpleasant experience such as witch-pricking. There are not many records of ordeals taking place during this period. It is likely that many of these procedures were not offically recorded. </b></div>',
+         footer: 'witches.is.ed.ac.uk',
+         confirmButtonText: 'Close',
+         type: 'info',
+         showCloseButton: true,
+       });
+     }
+   },
+   computed : {
+
+   },
+   mounted: function() {
+   },
+   layout: 'basic'
+ };
+
+</script>
+
+<style>
```

Comment:
I realised on this page, that because it is scrollable, the cookie banner floats in the middle of the page when you scroll. Are you able to fix this please? Thanks Ruby

Reply:
done, changed from absolute to fixed in the CookieBanner component